modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase

```diff
@@ -49,9 +49,43 @@ definition {
 	@description = "LPS-165727. Assert the details shown in the FDS table"
 	@priority = "4"
 	test HeadersCanBeLocalized {
+		task ("And Given: site URL localized to Spanish //localhost:8080/es/*") {
+			var portalURL = PropsUtil.get("portal.url");
 
-		// To-do HeadersCanBeLocalized Autom. test
+			Navigator.openSpecificURL(url = "${portalURL}/es/home");
 
+			Navigator.gotoPage(pageName = "Frontend Data Set Test Page");
+		}
```

Comment:
Hey Debs could you add `Navigator.gotoPage(pageName = "Frontend Data Set Test Page"); `here? When it changes the language of the page is being directed to home page and this is breaking test. I think could work

Reply:
before which command?